.github/ISSUE_TEMPLATE/test-week.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Schedule a Fedora Test Week
+about: Schedule a Fedora Test Week for a new Fedora major release
+title: ''
+labels: 'community', 'meeting'
+assignees: ''
+---
+
+## Initial Tasks
+
+[ ] Open this ticket in the `fedora-coreos-tracker` repo
+[ ] Open a ticket with [Fedora QA](https://pagure.io/fedora-qa/issues). See [F36 example](https://pagure.io/fedora-qa/issue/695)
+
+To be done after the Fedora QA folks have taken action on the QA ticket:
+
+[ ] Confirm the Test Day page is created on the Fedora Wiki.  For example: <https://fedoraproject.org/wiki/Test_Day:Fedora_36_CoreOS_2022-04-04>
+[ ] Confirm the Test Day results app is created. For example: <https://testdays.fedoraproject.org/events/131>
+[ ] Choose a day during the Test Week to host a video session for live debug help
+[ ] Setup a Google Meet or other video conference session
+[ ] Create a HackMD doc for capturing notes during live video session
+[ ] Find volunteers to enumerate new documentation + test cases requiired for Test Week
+[ ] File an issue on `fedora-coreos-tracker` with TODO items. For example: <https://github.com/coreos/fedora-coreos-tracker/issues/1147>
```

Comment:
Let's add another item here for requesting a fedora badge (like https://pagure.io/fedora-badges/issue/871)